albums/views.py

```diff
@@ -1,3 +1,122 @@
-from django.shortcuts import render
+import json
+import math
 
-# Create your views here.
+from django.http      import JsonResponse, HttpResponse
+from django.views     import View
+from django.db.models import Q
+
+from albums.models    import (Album, Genre, Artist, ReleaseType,
+                              PlayLink, AlbumPlayLink)
+
+class ListView(View): 
+    def get(self,request):
+        
+        query       = Q()
+        filter_set  = {}
+        page        = int(request.GET.get('page',1))
+        ALBUM_COUNT = 10
+        limit       = ALBUM_COUNT*page
+        offset      = limit - ALBUM_COUNT
```

Comment:
```suggestion
        limit       = ALBUM_COUNT * page
        offset      = limit - ALBUM_COUNT
```
아래 offset 해주신 것 처럼 변수와 연산자 사이는 띄우시는 것이 좋습니다

Reply:
넵 !